ui/src/services/utils.test.js

```diff
@@ -8,6 +8,16 @@ const testcases = [
   { storageCapacity: '1Mi' }
 ];
 
+const testcases2 = [
+  { storageCapacity: '42949670000' },
```

Comment:
Actually, the test case should be `42949670k`, because I do create the volume with `42949670000` but Kubernetes stores `42949670k`, and that's what you get in the table (so that's what we should be able to sort).